theano/sandbox/cuda/opt.py

```diff
@@ -1128,11 +1128,9 @@ def local_gpu_advanced_incsubtensor1(node):
             compute_capability = device_properties(active_device_no)['major']
             if (compute_capability < 2 or y.ndim != 2 or x.ndim != 2):
 
-                gpu_op = GpuAdvancedIncSubtensor1(
-                    set_instead_of_inc=set_instead_of_inc)
+                gpu_op = tensor.AdvancedIncSubtensor1(**node.op._props_dict())
```

Comment:
This explains the misunderstanding i had in previous comment. If i add `__props__` to the `GpuAdvancedIncSubtensor1` then the `_props_dict()` will use the correct `__props__`